### ISSUE TITLE
Hide sensitive information in the URL Query Params

### ISSUE THREAD

**Issue by camilamaia** [scanapi/scanapi#304]
## Hide sensitive information in the URL Query Params

Add a report configuration option to hide sensitive information in the url `query params`.

In the report, after the configuration, the URL should not be rendered like this

```
/1/members/me/boards?key=sadkjahs8731hdsh&token=sjdsajidu92837e8yd82327hd3278y
```

But instead, with the SENSITIVE_INFO label

```
/1/members/me/boards?key=SENSITIVE_INFO&token=SENSITIVE_INFO
```

For this issue, we only need to include a new option [here](https://github.com/scanapi/scanapi/blob/master/scanapi/hide_utils.py#L36) to handle "params" flow.

Please, see the [hiding_sensitive_information doc](https://scanapi.dev/docs_v1/configuration/hiding_sensitive_information.html) and https://github.com/scanapi/scanapi/issues/31 for better context.

**Comment by bottleInALightning** [comment]
Why dont you use a post request rather than get

**Comment by camilamaia** [comment]
@bottleInALightning sometimes the API was already built in this way. Example: [Trello API](https://developer.atlassian.com/cloud/trello/rest/api-group-actions/)

**Comment by aalphonsos** [comment]
Olá Camila. Gostaria de ficar responsável por essa issue. O contexto parece bem claro pra mim.

**Comment by marcuxyz** [comment]
> Olá Camila. Gostaria de ficar responsável por essa issue. O contexto parece bem claro pra mim.

Hi, Camila, I would like to be responsible for this issue. `LGTM`

**Comment by camilamaia** [comment]
@aalphonsos great, I've assigned you!

**Comment by camilamaia** [comment]
Closed by #325
